src/App.tsx

```diff
@@ -15,36 +17,109 @@
   'Garlic',
 ];
 
-export const App: React.FC = () => {
+const SORT_FIELD_ALPHABET: string = 'alphabet';
+const SORT_FIELD_LENGTH: string = 'length';
+const SORT_FIELD_RESET: string = '';
+
+type Props = {
+  sortField: string;
+  setSortField: () => string;
+  isReverse: string;
```

Comment:
[CODE KNOWLEDGE] - The `isReverse` prop should be of type `boolean` since it's used as a boolean value in the component's state and logic.